src/dmd/dtoh.d

```diff
@@ -321,6 +347,39 @@ public:
         this.buf = buf;
     }
 
+    private EnumKind getEnumKind(AST.Type type)
+    {
+        if (type) switch (type.ty)
+        {
+            case AST.Tint32:
+                return EnumKind.Int;
+            case AST.Tbool,
+                AST.Tchar, AST.Twchar, AST.Tdchar,
+                AST.Tint8, AST.Tuns8,
+                AST.Tint16, AST.Tuns16,
+                AST.Tuns32,
+                AST.Tint64, AST.Tuns64:
+                return EnumKind.Numeric;
+            default:
+                break;
+        }
+        return EnumKind.Other;
+    }
+
+    private void writeEnumTypeName(AST.Type type)
+    {
+        if (type.ty == AST.Tarray)
+        {
+            AST.TypeArray arr = cast(AST.TypeArray)type;
+            if (arr.next.ty == AST.Tchar)
```

Comment:
Do we care about wstring or dstring enums? Its fine to leave that for another PR.

Reply:
One step ahead of you... press F5.